### ISSUE TITLE
Game scoring bug

### ISSUE THREAD

**Issue by humuyan** [gcp/leela-zero-server#203]
http://zero.sjeng.org/match-games/5b739738cc3dde4a3e5c2909
In Game 101, the result shoud be `B+0.5`
Is it possible that the scoring system mistakes the winner?

**Comment by MartinDevelopment** [comment]
No, it is scored correctly since all stones that are not captured are considered alive. So the territories which still has the opponent's stones in it is considered 0 points. It doesn't matter because this way we still get the correct outcome.
